CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+
+cmake_minimum_required(VERSION 3.15)
+
+project(
+	saber
+	VERSION		1.0
+	DESCRIPTION	"Saber Utilities C++ Header Library")
+
+# Organizes project into folders to reduce clutter
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+
+include(GNUInstallDirs)
+
+# From ./.vscode/settings.json: "cmake.configureSettings"
+option(SABER_BUILD_TESTS "Build Unit Tests" OFF)
+option(SABER_BUILD_BENCHMARKS "Build Benchmarks" OFF)
+option(SABER_BUILD_DOCS "Build Doxygen Documentation" OFF)
```

Comment:
Ah, I see. These are the 3 CMake variables which control all of the test output. Either build tests, benchmarks, or documentation. It looks like only the Unit Test works right now, so I suppose the other 2 will come in a future PR.